src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -329,13 +330,15 @@ const ChatInfoScreen = () => {
               Created {moment(channelInfo?.createdAt).format('MM/DD/YYYY')}
             </Text>
           </View>
-          <TouchableOpacity onPress={handleOpenNameChange} style={styles.pencilIconTouchable}>
-            {isUpdatingName ? (
-              <ActivityIndicator />
-            ) : (
-              <IconPencil width={20} height={20} color={COLORS.gray400} />
-            )}
-          </TouchableOpacity>
+          {channelInfo?.channelType === CHANNEL_GROUP && (
+            <TouchableOpacity onPress={handleOpenNameChange} style={styles.pencilIconTouchable}>
+              {isUpdatingName ? (
+                <ActivityIndicator />
+              ) : (
+                <IconPencil width={20} height={20} color={COLORS.gray400} />
+              )}
+            </TouchableOpacity>
+          )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the conditional rendering of the pencil icon based on the channel type is a good UX decision. However, I would suggest moving this logic into the `handleOpenNameChange` function to keep the JSX cleaner.
```diff
- {channelInfo?.channelType === CHANNEL_GROUP && (
-   <TouchableOpacity onPress={handleOpenNameChange} style={styles.pencilIconTouchable}>
-     {isUpdatingName ? (
-       <ActivityIndicator />
-     ) : (
-       <IconPencil width={20} height={20} color={COLORS.gray400} />
-     )}
-   </TouchableOpacity>
- )}
+ <TouchableOpacity onPress={handleOpenNameChange} style={styles.pencilIconTouchable}>
+   {isUpdatingName ? (
+     <ActivityIndicator />
+   ) : (
+     <IconPencil width={20} height={20} color={COLORS.gray400} />
+   )}
+ </TouchableOpacity>
```
And then in your `handleOpenNameChange` function:
```diff
function handleOpenNameChange() {
+   if (channelInfo?.channelType !== CHANNEL_GROUP) {
+     return;
+   }
  // existing code...
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->